packages/trader/src/AppV2/Components/TradeParameters/Duration/wheelpicker.tsx

```diff
@@ -35,27 +34,24 @@ const DurationWheelPicker = observer(
                         duration_units_list.length == 1 && unit !== 'd',
                 })}
             >
-                <WheelPickerContainer
-                    data={
-                        selected_hour.length > 0 && selected_hour[0] == 24
-                            ? [
-                                  options[0],
-                                  [
-                                      {
-                                          value: 0,
-                                          label: <Localize i18n_default_text='0 min' />,
-                                      },
-                                  ],
-                              ]
-                            : options
-                    }
-                    defaultValue={[String(selected_time)]}
-                    containerHeight={handleContainerHeight()}
-                    inputValues={unit == 'h' ? selected_hour : selected_time}
-                    setInputValues={(index, val) => {
-                        setWheelPickerValue(index, val);
-                    }}
-                />
+                {unit !== 'h' ? (
```

Comment:
minor/nitpick: for future PRs - might be cool to avoid hardcoded strings like 'h', just have an enum or smth,

Reply:
done thanks